src/serverlist.h

```diff
@@ -171,13 +171,22 @@ class CServerListManager : public QObject
 
     QLocale::Country GetServerCountry() { return ServerList[0].eCountry; }
 
+    ESvrRegStatus GetSvrRegStatus() { return eSvrRegStatus; }
+
+    void SetSvrRegStatus ( ESvrRegStatus eStatus )
+    {
+        eSvrRegStatus = eStatus;
+        emit SvrRegStatusChanged();
```

Comment:
_May_ be able to stop the time here - but that seems to kick off `timeout()`.  So long as it doesn't make things loop...